docs/hello.rst

```diff
@@ -8,16 +8,10 @@ Depending on your hardware and OS, please execute one of the following commands:
 
 .. code-block:: bash
 
-  # Python 3.6+ needed
+  # python 3.6+ required
 
-  # CPU only. No GPU/CUDA needed. (Linux, OS X and Windows)
   python3 -m pip install taichi-nightly
 
-  # With GPU (CUDA 10.0) support (Linux only)
-  python3 -m pip install taichi-nightly-cuda-10-0
-
-  # With GPU (CUDA 10.1) support (Linux only)
-  python3 -m pip install taichi-nightly-cuda-10-1
```

Comment:
I removed them since I heard #714.

Reply:
I'll try to have v0.5.11 released with `taichi-nightly` covering all CUDA versions as soon as possible.